free-podcasts-screencasts-pt_BR.md

```diff
@@ -43,6 +43,9 @@
 
 * [Podquest](http://www.podquest.com.br) (podcast)
 
+### Go
+
+* [Aprenda Go](https://www.youtube.com/watch?v=WiGU_ZB-u0w&list=PLCKpcjBB_VlBsxJ9IseNxFllf-UFEXOdg) (screencast)
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

We also generally prefer to use the actual name from the resource, which includes an emote, this can be done with `:brazil:` :brazil:. Not sure if there're any rules around emotes, but I'm guessing it's fine.

```suggestion
* [Aprenda Go :brazil:](https://www.youtube.com/playlist?list=PLCKpcjBB_VlBsxJ9IseNxFllf-UFEXOdg) (screencast)
```
* [Aprenda Go :brazil:](https://www.youtube.com/playlist?list=PLCKpcjBB_VlBsxJ9IseNxFllf-UFEXOdg) (screencast)